### DISCUSSION TITLE
GitHub Student Developer Pack is still soon

### DISCUSSION THREAD

**Discussion by 4Fast** [community/community#136573] [GitHub Education]
### Hi everyone,

I applied for Developer Pack about 5 days ago and still waiting for benefits since article says wait about 5 days than create an topic on here. I wanted the create one. Can any Github Support help me about this issue?

**Comment by davevad93** [comment]
Hi @4Fast ,

Yes, as stated in the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) if you haven't received your benefits within 5 days you should raise a discussion here. However, most of the time the benefits become available on their own without the need for direct assistance from the education team.

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @4Fast was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849
